### ISSUE TITLE
Remove dependency on snap

### ISSUE THREAD

**Issue by bradwilson** [bradwilson/ansible-dev-pc#3]
**TL;DR: Remove dependency on snap so we can remove dependency on systemd in WSL 2 because bad things happen.**

---

The longer version:

I just spent a couple hours puzzling about why I couldn't run a Docker container that was exposing port 5000, because it said port 5000 was already in use. Using all the normal tools (`netstat`, `lsof`, `ss`, etc.) the port was shown as opened but not owned by any specific process.

There were actually three ports in this condition: `22` (for SSH), `53` (for DNS), and `5000` (which ostensibly is for UPnP, though I did not have UPnP enabled in my WSL image. This was very curious.

A second piece of the puzzle: I run multiple WSL distros. The distro where most of the ports were shown with owning processes (but 22, 53, and 5000) weren't we'll call image "A". So I fired up a shell for image "B", which it turns out was already running. I ran `sudo ss -plnt` and I saw a very strange thing: three ports were in use with named processes: 22, 53, and 5000; and a whole bunch of ports were active without owning processes.

The lists were identical but inverted.

It turns out WSL not only reflects open ports from WSL images into Windows (so you can run a server in WSL and talk to it from Windows), it also does that _between_ running WSL distros. So the open port 5000 in distro "B" was reflected into distro "A". (Coincidentally, I was running the same service in distro "B", but I've been trying to move myself into the new distro, and forgot that it was still running).

So why should ansible-dev-pc stop depending on snaps? Snaps require systemd. As noted above, enabling systemd means you start running at a minimum ssh & DNS. If you boot up two WSL distros at the same time that have systemd enabled, the second one to "boot" will collide with the first one.

And now you know.

<img src="https://media0.giphy.com/media/lRRjGTRlFwmQYFmmpU/200.gif" />

**Comment by bradwilson** [comment]
Everything snap-dependent has been removed except for Azure Storage Explorer, which is fine (for now) since it doesn't install on WSL (not really any point, since you can just run the Windows version instead).
